db/migrate/20160214032514_add_resource_type_id_to_item.rb

```diff
@@ -0,0 +1,14 @@
+class AddResourceTypeIdToItem < ActiveRecord::Migration
+  def up
+    if column_exists? :items, :resourcetype_id
+      remove_column :items, :resourcetype_id
```

Comment:
going with the assumption that the choice to not migrate the data is because there is no real data yet.